src/components/Modals/CourseMenu.vue

```diff
@@ -159,6 +184,32 @@ export default defineComponent({
     'edit-course-credit': (credit: number) => typeof credit === 'number',
   },
   methods: {
+    toggleDisplaySaveCourseModal() {
+      if (featureFlagCheckers.isSavedCoursesEnabled()) {
```

Comment:
Great job using feature flags! Better than me and Nidhi have been doing 💀

Reply:
LOL